libminifi/src/processors/PutFile.cpp

```diff
@@ -51,7 +51,9 @@ core::Property PutFile::CreateDirs(
     "Create Missing Directories",
     "If true, then missing destination directories will be created. "
         "If false, flowfiles are penalized and sent to failure.",
-    "true");
+    "true",
+    true,
```

Comment:
Before this property was not required but had a default value of true. Now it's required?

Reply:
It's required per documentation https://github.com/apache/nifi-minifi-cpp/blob/master/PROCESSORS.md#putfile. There are other properties out of sync with documentation/spec, which will be reconciled in future commits.

Reply:
Also note the case where a prop is required & has a default, and no value is given in the config is covered in unit tests. It will use the default value and validation will pass. So this change has no operational effect unless the user explicitly gives an empty value ("") in the config file.